models/imageDao.js

```diff
@@ -0,0 +1,49 @@
+const { appDataSource } = require("./datasource");
+
+//프로덕트 조회
+const checkProduct = async (productId) => {
+  return await appDataSource.query(`
+  SELECT * FROM products where id = ${productId}`);
+};
+
+//프로덕트아이디로 이미지 조회
+const getImage = async (productId) => {
+  return await appDataSource.query(`
+  SELECT * FROM images where product_id = ${productId}
+  `);
+};
```

Comment:
```suggestion
const getImageByProductId = async (productId) => {
  return await appDataSource.query(`
  SELECT * FROM images where product_id = ${productId}
  `);
};
```

- serviceLayer에서 imageDao.getImage(___) 로 호출하게 될텐데, 이 때 전달되는 인자가 무엇인지 명시되어 있지 않다면, image id로 잘못 오인될 가능성이 있습니다 :)